Containers/apache/Dockerfile

```diff
@@ -1,60 +1,40 @@
 # Caddy is a requirement
 FROM caddy:2.6.2-alpine as caddy
 
-FROM debian:bullseye-20221219-slim
+FROM httpd:2.4.54-alpine3.17
```

Comment:
the image needs to be change to the php-fpm-alpine

Reply:
why?

Reply:
ah sorry, wrong PR 😅